src/Capgemini.PowerApps.PackageDeployerTemplate/PackageTemplateBase.cs

```diff
@@ -342,7 +342,7 @@ public override bool AfterPrimaryImport()
                     this.PackageFolderPath);
 
                 this.MailboxSvc.UpdateApproveAndEnableMailboxes(this.TemplateConfig.Mailboxes
-                    .Where(m => m.EnvironmentPrefix == Environment.GetEnvironmentVariable(Constants.Settings.EnvironmentPrefix)));
+                    .Where(m => m.EnvironmentPrefix == this.GetSetting<string>(Constants.Settings.EnvironmentPrefix)));
```

Comment:
This is going in the right direction but it still doesn't align with existing features. Please see this comment - https://github.com/Capgemini/powerapps-packagedeployer-template/pull/41#discussion_r580368761